src/components/list/list.tsx

```diff
@@ -122,10 +126,23 @@ export class List {
             type: this.type,
             iconSize: this.iconSize,
         };
+        let maxLinesSecondaryText = +this.maxLinesSecondaryText?.toFixed();
+        if (this.maxLinesSecondaryText < 1) {
+            maxLinesSecondaryText = 1;
+        }
+
         const html = this.listRenderer.render(this.items, this.config);
 
         if (this.type !== 'menu') {
-            return html;
+            return (
+                <Host
+                    style={{
+                        '--maxLinesSecondaryText': `${maxLinesSecondaryText}`,
+                    }}
+                >
```

Comment:
Could this style maybe be added in the listRenderer.render method instead? 🤔

Reply:
@Befkadu1

Reply:
> Could this style maybe be added in the listRenderer.render method instead? 🤔

Why? The objective is to style `limel-list` tag as you see in the pic below
If you can style the `limel-list` tag from listRenderer.render, you are welcome to do. But I wonder why we do it in listRenderer.render method

<img width="463" alt="Screenshot 2021-04-20 at 18 11 16" src="https://user-images.githubusercontent.com/22968497/115429679-ebd2e280-a203-11eb-8d44-d89140ef7bd7.png">

Reply:
I thought the objective was to override the style of the .mdc-list-item__secondary-text. Therefore, I thought the styling could be added to the ul tag instead, which is rendered in the listRenderer.render method. But I don't know what is the best practice when styling. 😛

Reply:
Please note that we are not really "styling" in that sense. We are applying a css variable (`--maxLinesSecondaryText`) to the parent (as a style which we have to), so that it becomes available for the children to use. So I think it's correct to style it at that level.

Reply:
Okey then I approve it! 🙂

Reply:
Thanks Malin. So I'll squash the fixup now :)

Reply:
now somebody should rebase and merge :)